libportability-gfx/src/impls.rs

```diff
@@ -4107,6 +4107,14 @@ pub extern "C" fn gfxCreateWin32SurfaceKHR(
             VkResult::VK_SUCCESS
         }
     }
+    #[cfg(feature = "gfx-backend-dx11")]
```

Comment:
We could merge the surface creation with dx12 unless we expect these to diverge

Reply:
indeed they look identical